configure.ac

```diff
@@ -46,7 +46,6 @@ fi
 AC_PROG_CPP
 AC_PROG_MAKE_SET
 AC_PROG_INSTALL
-AC_PROG_MKDIR_P
```

Comment:
We use `$(MKDIR_P)` in a couple of Makefiles.
The build works without them (for me).
Are you sure it's safe for all users?